packages/patternfly-4/react-integration/demo-app-ts/src/components/demos/DropdownDemo/DropdownDemo.tsx

```diff
@@ -62,13 +62,14 @@ export class DropdownDemo extends React.Component<{}, DropdownState> {
         isActionOpen
       });
     };
-    this.onActionSelect = event => {
+    this.onActionSelect = () => {
```

Comment:
Should we restore the `event` props in the functions above? Perhaps just rename it as `_event` to avoid the warning?